### PR TITLE
Add Node 18

### PR DESCRIPTION
**Pull request by tylerodonnell**
Adds Node 18 Buster and Bullseye images.

## New Image Checklist

If you're adding a new image, make sure you have done the following.

* [x] Added to lint workflow matrix (`.github/workflows/lint.yml`)
* [x] Added to build workflow matrix (`.github/workflows/build.yml`)
* [x] Added to Makefile (`Makefile`)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -23,6 +23,8 @@ jobs:
           - 16-stretch-slim
           - 16-buster-slim
           - 16-bullseye-slim
+          - 18-buster-slim
```
Comment (mloberg) [inline .github/workflows/build.yml:26]: What do we think about only providing a bullseye build?

Comment (tylerodonnell) [reply .github/workflows/build.yml:26]: I'm not opposed to that idea, and it forces folks to be on the latest version of Debian. I can't think of anything that _wouldn't_ work on Bullseye.

Comment (mloberg) [reply .github/workflows/build.yml:26]: Let's do it. If we find something that isn't working, we can always add Buster at that time

Comment (tylerodonnell) [reply .github/workflows/build.yml]: Sounds good!

**Review by mloberg — APPROVED** [review]

**Pull request merged**